public/partials/calculations/calc-data-inventory.html

```diff
@@ -0,0 +1,34 @@
+<div ng-controller="CalcController" class="inventory-holder">
+  <h1 class="heading">Manage Calculations</h1>
+  <select ng-change="inventoryChanged()" ng-model="selectedInventory">
+    <option ng-repeat="inventory in inventories" value="{{inventory._id}}">{{inventory.in_inventory_desc}} ({{inventory.in_start_date | date:'longDate'}} - {{inventory.in_end_date | date: 'longDate'}})</option>
+  </select>
+  <select ng-change="sectorChanged()" ng-model="selectedSector">
+    <option ng-repeat="sector in sectors" value="{{sector._id}}">{{sector.se_name}}</option>
+  </select>
+  <div class="horizline"></div>
+
+  <div ng-show="isAvailable">
+    <a class="smaller add-category-button w-inline-block" data-ix="add-category-modal" ng-click="getAllEF(selectedInventory, selectedSector)">
+      <div>Load Emission Factors</div>
+    </a>
+  </div>
+  <br />
+
+  <div ui-tree data-drag-enabled="false" data-nodrop-enabled="false">
+    <ol ui-tree-nodes="" ng-model="activity_data">
+      <li ng-repeat="item in activity_data" ng-if="item.issues.length == 0" ui-tree-node>
+        <div layout="row" ui-tree-handle>
+          <span flex>AD ({{item.ca_category.ca_code_name}})</span>
+          <span flex>{{item.da_data_value | number}} {{item.un_unit.un_unit_symbol}}</span>
+        </div>
+        <ol ui-tree-nodes="" ng-model="matches">
+          <li ng-if="subItem.issues.length == 0" ng-repeat="subItem in getAssociatedInfo(item, emission_factors)" ui-tree-node>
+            <calc-item sub-item="subItem" activity="item" selected="false"></calc-item>
```

Comment:
So a few things:
<img width="768" alt="screen shot 2018-02-05 at 6 54 52 pm" src="https://user-images.githubusercontent.com/1425164/35835612-acfbe7fe-0aa9-11e8-9b62-76c9558223b9.png">
The units being displayed is actually incorrect (highlighted in red).  This should show `kg` and not `kg/TJ`.  The platform should have a combobox that allows the selection of the units for the final calculation.  This is necessary for the final value output in the report.

Reply:
The combobox also needs some alignment (from a design perspective)